The regular expression `*== 1 failed in * ==*` from
test_modeltest.py is not designed to handle warnings.

--- a/tests/test_modeltest.py
+++ b/tests/test_modeltest.py
@@ -351,6 +351,7 @@
         """
         [pytest]
         qt_log_level_fail = NO
+        addopts = -p no:warnings
     """
     )
     testdir.makepyfile(
